examples/resources/juju_jaas_access_model/resource.tf

```diff
@@ -0,0 +1,7 @@
+resource "juju_jaas_access_model" "development" {
+  model_uuid       = juju_model.development.id
```

Comment:
Use uuid rather than id, as it'll be available before 0.15.0 goes out the door.

Reply:
Done